lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.ts

```diff
@@ -242,6 +259,30 @@ export class ContentMetadataComponent implements OnChanges, OnInit, OnDestroy {
         }
     }
 
+    saveEditChanges(buttonType: string, event: Event, group?: any) {
+        this.saveChanges(event);
+        
```

Comment:
I see many trailing spaces issues in that file, please correct all of them: 
<img width="800" alt="image" src="https://github.com/Alfresco/alfresco-ng2-components/assets/115619721/88932a5e-d681-4f28-b9e4-49dfd6d62fdc">